tests/integration/test_fabric_extractor.py

```diff
@@ -110,3 +136,26 @@ def test_extractor_timeseries_service(cognite_client, raw_time_series, test_extr
     for external_id, group in raw_time_series.groupby("externalId"):
         group = prepare_test_dataframe_for_comparison(group)
         assert_data_points_df_in_cdf(external_id, group, cognite_client)
+
+
+def test_extractor_abfss_file_upload(
+    cognite_client, test_csv_file_path, test_extractor, azure_credential
+):
+    # Upload the file to Fabric
+    (
+        upload_file_to_lakehouse(
+            TEST_FILE_NAME,
+            test_csv_file_path,
+            test_extractor.config.source.abfss_prefix,
+            test_extractor.config.source.file_path,
+            azure_credential,
+        ),
+    )
+    # Run extractor upload
+    test_extractor.upload_files_from_abfss(
```

Comment:
Why don't you run the extractor itself here rather than just the upload_files_from_abfss method?

Reply:
In this test I just want to focus on the file upload scenario, we can ensure that the conditional paths in the run method are occurring correctly in a unit test.